### DISCUSSION TITLE
Omit `--name` flag with `nargo init`

### DISCUSSION THREAD

**Discussion by critesjosh** [noir-lang/noir#3003] [Ideas]
### Problem

Can we just do `nargo init [name]` instead of requiring `nargo init --name [name]`? Its a bit more intuitive and easier to type. Not sure if there are any downsides.

### Happy Case

It's easier to create a Noir project.

### Would you like to submit a PR for this Issue?

No

**Comment by Savio-Sou** [comment]
Should be an easy task for the team / contributors to pick up; converted into https://github.com/noir-lang/noir/issues/3007.
